src/components/compress/custom-els/MultiPanel/index.ts

```diff
@@ -1,19 +1,75 @@
-import './styles.css';
+import * as style from './styles.css';
+import { transitionHeight } from '../../../../lib/util';
 
-function getClosestHeading(el: Element) {
-  const closestEl = el.closest('multi-panel > *');
-  if (closestEl && closestEl.classList.contains('panel-heading')) {
-    return closestEl;
+interface CloseAllOptions {
+  exceptFirst?: boolean;
+}
+
+const openOneOnlyAttr = 'open-one-only';
```

Comment:
New feature in multi-panel. This attribute ensures a maximum of zero panels can be open at once.